<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>222</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modbus Slave</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/img/Bus.png</normaloff>:/img/Bus.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_settings">
      <property name="minimumSize">
       <size>
        <width>340</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>340</width>
        <height>40</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblModbusMode">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Modbus Mode</string>
         </property>
         <property name="buddy">
          <cstring>cmbModbusMode</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbModbusMode">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>RTU</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TCP</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSlaveID">
         <property name="text">
          <string>Slave ID</string>
         </property>
         <property name="buddy">
          <cstring>sbSlaveID</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbSlaveID">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>31</width>
           <height>9</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_commands">
      <property name="minimumSize">
       <size>
        <width>340</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>340</width>
        <height>40</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btStartStop">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Start-Stop</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
         <property name="icon">
          <iconset resource="icons/pyModSlaveQt.qrc">
           <normaloff>:/img/run-16.png</normaloff>
           <normalon>:/img/stop-16.png</normalon>:/img/run-16.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSimCycle">
         <property name="text">
          <string>Simulate Every (msec)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spInterval">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="minimum">
          <number>1000</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>500</number>
         </property>
         <property name="value">
          <number>5000</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>26</width>
           <height>9</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>384</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSerial_RTU"/>
    <addaction name="actionTCP"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionIO_Data"/>
    <addaction name="actionBus_Monitor"/>
    <addaction name="separator"/>
    <addaction name="actionReset_Counters"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuData"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/close16.png</normaloff>:/img/close16.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSerial_RTU">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/edit-16.png</normaloff>:/img/edit-16.png</iconset>
   </property>
   <property name="text">
    <string>Modbus RTU...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionTCP">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/network-16.png</normaloff>:/img/network-16.png</iconset>
   </property>
   <property name="text">
    <string>Modbus TCP...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/info16.png</normaloff>:/img/info16.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/options-16.png</normaloff>:/img/options-16.png</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCoils">
   <property name="icon">
    <iconset>
     <normaloff>:/img/DO.png</normaloff>:/img/DO.png</iconset>
   </property>
   <property name="text">
    <string>Coils</string>
   </property>
  </action>
  <action name="actionDigital_Inputs">
   <property name="icon">
    <iconset>
     <normaloff>:/img/DI.png</normaloff>:/img/DI.png</iconset>
   </property>
   <property name="text">
    <string>Digital Inputs</string>
   </property>
  </action>
  <action name="actionHolding_Registers">
   <property name="icon">
    <iconset>
     <normaloff>:/img/AO.png</normaloff>:/img/AO.png</iconset>
   </property>
   <property name="text">
    <string>Holding Registers</string>
   </property>
  </action>
  <action name="actionAnalog_Inputs">
   <property name="icon">
    <iconset>
     <normaloff>:/img/AI.png</normaloff>:/img/AI.png</iconset>
   </property>
   <property name="text">
    <string>Analog Inputs</string>
   </property>
  </action>
  <action name="actionIO_Data">
   <property name="icon">
    <iconset>
     <normaloff>:/img/IO.png</normaloff>:/img/IO.png</iconset>
   </property>
   <property name="text">
    <string>IO Data</string>
   </property>
  </action>
  <action name="actionBus_Monitor">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/view-16.png</normaloff>:/img/view-16.png</iconset>
   </property>
   <property name="text">
    <string>Bus Monitor</string>
   </property>
  </action>
  <action name="actionReset_Counters">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/trash-16.png</normaloff>:/img/trash-16.png</iconset>
   </property>
   <property name="text">
    <string>Reset Counters</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spInterval</tabstop>
 </tabstops>
 <resources>
  <include location="icons/pyModSlaveQt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
